<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitiativesWidget</class>
 <widget class="QWidget" name="InitiativesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySim Physical Porperty</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="InitiativesInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="title">
      <string>Initiatives input</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="InitiativeNameLabel">
        <property name="text">
         <string>Initiative name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="InitiativeName">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="UnitLabel">
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="Unit">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="MainValueLabel">
        <property name="text">
         <string>Mainbody value</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Gui::InputField" name="MainbodyValue">
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ExtraValueLabel">
        <property name="text">
         <string>Extra value</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="Gui::InputField" name="ExtraValue"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
